swagger.yml

```diff
@@ -167,6 +192,21 @@ paths:
           description: Success, return response
           schema:
             $ref: '#/definitions/CheckAllResults'
+  /cluster_health:
+    get:
+      description: Checks the full graph. Returns a binary OK or not OK.
+      produces:
+        - application/json
+      operationId: clusterHealth
+      responses:
+        200:
+          description: Healthy cluster
+          schema:
+            $ref: '#/definitions/ClusterHealthResults'
+        503:
```

Comment:
I am debating what the error response code should be... And should it even be a `5xx` code (server side error) or a `4xx` (client error) - the problem is there are no good `5xx` status codes that capture this situation and if we rephrase the question as "The client expects everything to be ok and is checking that", then we can return one of:
- [406 Not Acceptable](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406)
- [409 Conflict](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409)

But again this is trying to apply HTTP semantics to a slightly different problem and I am probably splitting hairs here...

Reply:
I had the same thought last night - something in the 400 range for this would be my preference so that it can be distinguished from an actual server-side error in Goldpinger. I originally imagined 418 because it's pretty unlikely anything is ever going to throw that. :v

Reply:
I always have a soft spot for 418, so 418 it is!